gnovm/tests/files/interface27b.gno

```diff
@@ -7,4 +7,4 @@ func main() {
 }
 
 // Output:
-// map{(0 int):(undefined)}
```

Comment:
Shouldn't type conversion make this `(undefined) error`?

Reply:
thanks for identify this, it is because by doing the split work, this kind of conversion(untyped nil -> typed nil) is happened in #1891. see: https://github.com/gnolang/gno/blob/953765cb3bc17f6212f6e96ee8d138fea765beca/gnovm/tests/files/interface27b.gno

Reply:
this is reverted now that it's not necessary in this PR.